src/popup/pages/AccountDetail/index.tsx

```diff
@@ -101,15 +115,49 @@ const TransactionList: SFC<any> = observer(({ classes, store: { accountDetailSto
   </div>
 ));
 
-const TokenList: SFC<any> = observer(({ classes, store: { accountDetailStore: { tokens } } }: any) =>
-  tokens.map(({ name, symbol, balance }: QRCToken) => (
-    <ListItem divider key={symbol} className={classes.listItem}>
-      <div className={classes.tokenInfoContainer}>
-        <Typography className={classes.tokenName}>{name}</Typography>
-      </div>
-      <AmountInfo classes={classes} amount={balance} token={symbol} convertedValue={0} />
-    </ListItem>
-  )));
+const TokenList: SFC<any> = observer(({ history, classes,
+  store: { accountDetailStore, accountDetailStore: { tokens } } }: any) => (
+  <div>
+    {tokens.map(({ name, symbol, balance }: QRCToken, index: number) => (
+      <ListItem divider key={symbol} className={classes.listItem}
+        onClick = {() => accountDetailStore.editTokenMode && accountDetailStore.removeTokenAtIndex(index)}
+      >
+        {accountDetailStore.editTokenMode &&
+          <Button
+            className={classes.deleteButton}
+            id="removeTokenButton"
+          >
+            <ReactSVG path="images/Ic_detele.svg" />
```

Comment:
we could do this with `Material-UI/icons` as well. did you just find this off the internet or was it provided by Lili? if its just one you found off the internet, I would prefer to stick with the Material UI provided icons to keep consistency.

also, the name should be `ic_delete.svg` im guessing.

Reply:
I got it from lili - she had me export it from figma

Reply:
Cool. Just rename then.